doc/lsst.obs.decam/using-obs-decam.rst

```diff
@@ -0,0 +1,79 @@
+.. _lsst.obs.decam.gen3:
+
+#################
+The Gen3 workflow
+#################
+
+These directions are for users who wish to create their own data repositories independent of any Rubin/LSST-supported shared repositories.
+
+Create a Gen3 data repository and write curated calibration products to it (these include crosstalk, linearity, and defects). The latter step creates the `DECam/calib` collection.
+
+.. prompt:: bash
+
+   cd /path/to/desired/repo
+   butler create .
+   butler register-instrument . lsst.obs.decam.DarkEnergyCamera
+   butler write-curated-calibrations . lsst.obs.decam.DarkEnergyCamera
+
+Ingest raw science and calibration (bias and flat) frames, which creates the `DECam/raw/all` collection. Run overscan correction on all the raws so they may be used as crosstalk sources during ISR.
```

Comment:
I don't see overscan correction in any of butler/pipetask lines below?

Reply:
The overscan correction is the only part of ISR that happens in `RunIsrForCrosstalkSources.yaml`. I will clarify.